src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerOptions.cs

```diff
@@ -34,6 +35,12 @@ public class KestrelServerOptions
         public bool UseTransportThread { get; set; }
 
         /// <summary>
+        /// Gets or sets a value that determines how Kestrel should schedule user callbacks.
+        /// </summary>
+        /// <remarks>The default mode is <see cref="SchedulingMode.ThreadPool"/></remarks>
+        public SchedulingMode ApplicationSchedulingMode { get; set; } = SchedulingMode.ThreadPool;
```

Comment:
Yea I'll just do that since I made default and thread pool the same in this PR. Originally I was going to make Default throw